scopes/workspace/workspace/scope-subcommands/scope-set.cmd.ts

```diff
@@ -1,23 +1,22 @@
 import { Command } from '@teambit/cli';
-import { PATTERN_HELP } from '@teambit/legacy/dist/constants';
+import { PATTERN_HELP, COMPONENT_PATTERN_HELP } from '@teambit/legacy/dist/constants';
 import chalk from 'chalk';
 import { Workspace } from '../workspace';
 
 export class ScopeSetCmd implements Command {
   name = 'set <scope-name> [component-pattern]';
   description =
-    'Sets components with a default-scope. If no component is specified, sets the workspace with a default scope';
+    'Sets the scope for specified component/s. If no component is specified, sets the default scope of the workspace';
   arguments = [
-    { name: 'scope-name', description: 'the scope name to use as the default scope' },
+    { name: 'scope-name', description: 'name of the scope to set' },
     {
       name: 'component-pattern',
-      description:
-        'component name, component id, or component pattern. use component pattern to select multiple components. \nuse comma to separate patterns and "!" to exclude. e.g. "ui/**, !ui/button"\nwrap the pattern with quotes',
+      description: COMPONENT_PATTERN_HELP,
     },
   ];
   options = [];
   group = 'development';
-  extendedDescription = `default scopes for components are set in the bitmap file. the default scope for a workspace is set in the workspace.jsonc. a component is set with a scope (as oppose to default scope) only once it is versioned.'
+  extendedDescription = `default scopes for components are set in the bitmap file. the default scope for a workspace is set in the workspace.jsonc.\n a component is set with a scope (as oppose to default scope) only once it is versioned.'
```

Comment:
Please remove the space after `\n` otherwise, it'll indent with one space in the new line

Reply:
done